theme/ItaliaTheme/Subsites/ItaliaTheme/Blocks/_info.scss

```diff
@@ -0,0 +1,37 @@
+@use '../../all_variables' as *;
+
+@mixin info($subsite-primary, $subsite-light-theme) {
+  .public-ui,
+  &.public-ui {
+    .infoblock {
+      .bg-alert-primary,
+      .bg-alert-secondary,
+      .bg-alert-tertiary {
+        &.bg-color-true {
+          .draftjs-buttons {
```

Comment:
![image](https://github.com/RedTurtle/design-comuni-plone-theme/assets/116291154/6bc45a38-b99f-45b7-be6a-ee98db8fec60)

Bisogna aggiustare gli stili dell'icona del blocco, il colore del bottone se lo sfondo del blocco non è colorato, e l'hover del bottone quando lo sfondo è primary e colorato

Reply:
Questo sarà fatto in un'altra PR